dydx/dydxViews/dydxViews/_v4/TakeProfitStopLoss/dydxTakeProfitStopLossViewModel.swift

```diff
@@ -61,24 +71,61 @@ public class dydxTakeProfitStopLossViewModel: PlatformViewModel {
         }
     }
 
+    private func createCta(parentStyle: ThemeStyle, styleKey: String?) -> AnyView? {
+        let buttonText: String
+        let buttonState: PlatformButtonState
+        switch submissionReadiness {
+        case .readyToSubmit:
+            buttonText = DataLocalizer.shared?.localize(path: "APP.TRADE.ADD_TRIGGERS", params: nil) ?? ""
```

Comment:
We'd also need an "Edit Trigger" in addition to "Add Trigger".

Reply:
will add